.azure_pipelines/olive-ci.yaml

```diff
@@ -85,10 +87,81 @@ jobs:
 
 - template: job_templates/olive-build-template.yaml
   parameters:
-    name: Linux_GPU_CI
+    name: Linux_GPU_CI_latest_ORT
     pool: $(OLIVE_POOL_UBUNTU2004)
     windows: False
     device: gpu
+    onnxruntime: onnxruntime-gpu
+    examples:
+      bert_cuda_gpu:
+        exampleFolder: bert
+        exampleName: bert_cuda_gpu
+
+- template: job_templates/olive-build-template.yaml
```

Comment:
I think this is too many tests to run for a single CI run. Might be better for us to have some daily or weekly scheduled pipeline runs for the previous version and latest nightly. 

For CI we can just run the latest like we usually do.

Reply:
yes, run daily or weekly against ORT nightly is fine. For ORT 1.15.1, I am not sure we should enable it on each PR. 
From the time being, the run is parallel so the total waiting time is same with before.

Reply:
separate the CI pipelines